config/env_schema.hjson

```diff
@@ -70,9 +70,16 @@
         "MIVIDEO": {
             "type": "object",
             "properties": {
-                "service_account_json_filename": {"type": "string"}
+                "default_last_timestamp": {"type": "string", "format": "date-time"},
+                "udp_service_account_json_filename": {"type": "string"},
+                "kaltura_partner_id": {"type": "integer"},
+                "kaltura_user_secret": {"type": "string"},
```

Comment:
True.  But I like it there and at the end of Python data structures, too.  It may prevent an error or two from someone forgetting to add a comma when they add a new line.

Reply:
Just to be clear, I'm not asking or requiring you to change this. In my own usage (unless I make a mistake), I choose to only include commas if they are necessary and meaningful and not worry about future-proofing. I will leave what to do with this up to you.